v0/util/index.js

```diff
@@ -1151,6 +1151,25 @@ function isCdkDestination(event) {
   );
 }
 
+function verifyDynamicFormAttribute(eventArray) {
```

Comment:
It's not a good candidate to put in the common utils as it specifically checks for `from` and `to` fields.